dask_expr/concat.py

```diff
@@ -84,8 +84,23 @@ def _simplify_down(self):
         )
 
     def _simplify_up(self, parent):
-        # TODO: implement passing projections through
-        return
+        if isinstance(parent, Projection):
+            columns = parent.columns
+            columns_frame = [
+                sorted(set(frame.columns).intersection(columns))
+                for frame in self._frames
+            ]
+            if all(
+                cols == frame.columns
```

Comment:
can we also avoid the projection pushdown if the columns are the same, but in a different order?

Reply:
Yep I’ll adjust these conditions

Reply:
Added as well